recipes/tinker/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "tinker" %}
+{% set version = "8.5.3" %}
+{% set sha256 = "96ede325ce32a1a330d1e9e62258b089a9a8563769b16eb42705e6415355ed86" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://dasher.wustl.edu/tinker/downloads/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - cd source
+    - make TINKERDIR=$PREFIX RANLIB=ranlib LIBDIR="-L. -L$PREFIX/lib/linux" F77=$GFORTRAN FFTWDIR="$PREFIX"  # [linux]
+    - make TINKERDIR=$PREFIX F77=$GFORTRAN FFTWDIR="$PREFIX" LINKFLAGS='$(OPTFLAGS) -static-libgcc -headerpad_max_install_names'  # [osx]
+    - make install TINKERDIR=$PREFIX
+
+requirements:
+  build:
+    - {{ compiler("fortran") }}
+    - make
+  host:
+    - fftw
+  run:
+    - openmp
+
+test:
+  commands:
+    - which alchemy
```

Comment:
This test doesn't seem related to the package

Reply:
I've added the test commands.